content/pages/index/onboarding/steps/ready.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Good job! You are ready to explore the portal, scroll down to see more.",
+  "suggestion": "One last thing: we are never going to ask for your private key,\n\n remember to never share it with anyone!",
```

Comment:
nitpick(blocking): do not couple the variable name too much to the content. It's a body text after all.